### PR TITLE
[POWERCFG] Add file hdr. Improve pt-PT translation.  Fix accel collisions. Add missing accels keys.

### PR DESCRIPTION
**Pull request by zecarlos1957**
Add file header
Improve Portuguese (pt-PT) translation
Remove accell from Ok and Cancel buttons.
Fix accel collisions. Add missing accels keys.

### CODE REVIEW

File: dll/cpl/powercfg/lang/pt-PT.rc [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     ReactOS Power Configuration Applet
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2019-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```
Comment (binarymaster) [inline dll/cpl/powercfg/lang/pt-PT.rc:5]:
```suggestion
 * TRANSLATOR:  Copyright 2020-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```

File: dll/cpl/powercfg/lang/pt-PT.rc [hunk 2]
```diff
@@ -90,7 +97,7 @@ BEGIN
     ICON IDI_ACDC, IDI_ACDC, 7, 7, 0, 0
     LTEXT "Seleccione as definições de poupança de energia que deseja utilizar.", -1, 37, 6, 208, 20
     GROUPBOX "Opções", -1, 6, 30, 240, 65
-    AUTOCHECKBOX "Mostrar sempre o &icone na barra de tarefas", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
+    AUTOCHECKBOX "Mostrar sempre o &ícone na barra de tarefas", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
```
Comment (HBelusca) [inline dll/cpl/powercfg/lang/pt-PT.rc:100]: Is this an "easy" keyboard accelerator to use? (using accentued i)

File: dll/cpl/powercfg/lang/pt-PT.rc [hunk 3]
```diff
@@ -159,11 +166,11 @@ BEGIN
     IDS_TIMEOUT8 "Após 25 mins"
     IDS_TIMEOUT9 "Após 30 mins"
     IDS_TIMEOUT10 "Após 45 mins"
```
Comment (HBelusca) [inline dll/cpl/powercfg/lang/pt-PT.rc:168]: is "mins" OK in portuguese?

**Review by HBelusca — APPROVED** [review]

File: dll/cpl/powercfg/lang/pt-PT.rc (hunk 3 above)
Comment (zecarlos1957) [reply dll/cpl/powercfg/lang/pt-PT.rc:168]: Is ok.

File: dll/cpl/powercfg/lang/pt-PT.rc (hunk 2 above)
Comment (zecarlos1957) [reply dll/cpl/powercfg/lang/pt-PT.rc]: you're right.

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**